src/app/user/user.controller.ts

```diff
@@ -0,0 +1,90 @@
+import { JwtAuthGuard } from "@/app/auth/guards/jwt-auth.guard";
+import { FilterUserListDto } from "@/app/user/dto/filter-user-list.dto";
+import { UpdateUserDto } from "@/app/user/dto/update-user.dto";
+import { User } from "@/app/user/entities/user.entity";
+import { UserService } from "@/app/user/user.service";
+import { GetUser } from "@/decorators/get-user.decorator";
+import { PublicFileValidatorInterceptor } from "@/interceptors/public-file-validator.interceptor";
+import {
+  Body,
+  Controller,
+  DefaultValuePipe,
+  Delete,
+  Get,
+  Param,
+  ParseIntPipe,
+  Patch,
+  Post,
+  Query,
+  UploadedFile,
+  UseGuards,
+  UseInterceptors,
+} from "@nestjs/common";
+import { FileInterceptor } from "@nestjs/platform-express";
+
+@Controller("user")
```

Comment:
should have a constant file for all route in app
st like
Routes = {
    user: '/user',
    file: '/file'
}